jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -1263,6 +1263,24 @@ ecma_string_get_length (const ecma_string_t *string_p) /**< ecma-string */
 } /* ecma_string_get_length */
 
 /**
+ * Checks whether is ecma string empty or not
+ *
+ * @return true  - if empty
+ *         false - otherwise
+ */
+bool
```

Comment:
I tried, but didn't work well. On sunspider tests this patch does not gain any performance, because the changes are slightly covered by the benchmark. It's more like a style style and binary size improvement.